src/components/quickSearchPage/index.js

```diff
@@ -159,7 +158,11 @@ const FoamTreeSearchPage = () => {
       setError(true);
       return;
     }
-    const docs = data;
+    const docs = data.map((doc) => {
+      const newDoc = Object.assign({}, doc);
+      newDoc.keywords = doc.keywords.toLowerCase().split(';').map(keyword => keyword.split(',')).reduce((semicolon, coma) => [...semicolon, ...coma], []);
```

Comment:
line is too long, split

Reply:
Better names: function accepts `currentItem, aggregation`

Reply:
@ivankozlovcodes fixed